sdk/appconfiguration/azure-appconfiguration/samples/sync_token_samples.py

```diff
@@ -12,13 +12,12 @@
     This sample demos update_sync_token for the AzureAppConfigurationClient
 USAGE: python sync_token_samples.py
 """
-
+from typing import Any, List
 from azure.appconfiguration import AzureAppConfigurationClient
-from util import print_configuration_setting, get_connection_string
+from util import get_connection_string
 
 
-def handle_event_grid_notifications(event_grid_events):
-    # type: (List[dict[str, Any]]) -> None
+def handle_event_grid_notifications(event_grid_events: List[dict[str, Any]]) -> None:
```

Comment:
I believe you'll want to use the `Dict` type from `typing` here instead of `dict`. `dict` is interpreted as a general type in Python 3.9+, but in older versions `Dict` is usually what you'll want